s3prl/downstream/mos_prediction/config.yaml

```diff
@@ -0,0 +1,39 @@
+runner:
+  total_steps: 20000
+  gradient_clipping: 5
+  gradient_accumulate_steps: 2
+
+  log_step: 100
+  eval_step: 250
+  save_step: 500
+  max_keep: 1
+  eval_dataloaders:
+    - dev
+    - vcc2018_test
+    - vcc2016_test
+
+optimizer:
+  name: Adam
+  lr: 1.0e-4
+
+# comment the whole scheduler config block
+# to disable learning rate scheduling
+scheduler:
+  name: linear_schedule_with_warmup
+  num_warmup_steps: 500
+
+downstream_expert:
+  datarc:
+    num_workers: 8
+    train_batch_size: 8
+    eval_batch_size: 12
+    vcc2018_file_path: /home/raytz/Disk/VCC_2018
```

Comment:
Can we add some description on the README page for the link of these dataset?
Also the expected directory structure after people download.
If you need further preprocessing after downloading, please consider adding a shell script to automatically prepare the downstream dataset environment.